includes/pages/admin_arrive.php

```diff
@@ -19,48 +19,51 @@ function admin_arrive()
     $msg = '';
     $search = '';
     $request = request();
+    $admin_arrive = auth()->can('admin_arrive');
 
     if ($request->has('search')) {
         $search = strip_request_item('search');
         $search = trim($search);
     }
 
     $action = $request->get('action');
```

Comment:
Can too be evaluated inside the if

Reply:
-v

Reply:
yes I could add it to both if and else if

Reply:
move down would've helped me more

Reply:
I meant the `action` part but yeah